release-notes.hbs.md

```diff
@@ -367,6 +367,12 @@ This release has the following security fixes, listed by component and area.
 
 The following issues, listed by component and area, are resolved in this release.
 
+#### <a id='1-6-0-scst-store-ri'></a> Supply Chain Security Tools - Store
+- Implemented basic logging in the AMR database.
```

Comment:
Just double-checking, are we using the AMR short-form/acronym in official docs?

Reply:
In the first mention of Artifact Metadata Repository (AMR) on a page, we should use the long form. Then subsequent mentions of it on the page can just use AMR.

Reply:
A good catch Debbie. Fortunately, someone in the `New Features` section already mentions AMR, which is before this one. So I think it's fine to use the short form at this point